app/updates/BreakingNewsUpdate.scala

```diff
@@ -39,55 +38,66 @@ class BreakingNewsUpdate(val config: ApplicationConfiguration, val ws: WSClient,
     collection: ClientHydratedCollection,
     email: String
   ): Future[Result] = {
+    // TODO MRB:
+    // check we still log in the correct places
+    // does putting it in the web container set the "alert" flag?
+
     structuredLogger.putLog(LogUpdate(HandlingBreakingNewsCollection(collectionId), email))
 
-    val futurePossibleErrors = Future.traverse(collection.trails)(trail => sendAlert(trail, email, collectionId).map(trail -> _))
-    futurePossibleErrors.map { listOfPossibleErrors => {
-      val errors = listOfPossibleErrors.collect { case(trail, Some(error)) => trail -> error }
-      if (errors.isEmpty) {
-        Ok
-      } else {
-        errors.foreach { case (trail, error) =>
-          logger.error(s"Error sending breaking news. Returning to client: $error. Trail: $trail")
-        }
+    val requests = createRequests(collection.trails, email)
 
-        InternalServerError(Json.toJson(errors.map { case(_, error) => error }))
-      }
-    }}
-  }
+    Future.traverse(requests) { request =>
+      structuredLogger.putLog(LogUpdate(HandlingBreakingNewsTrail(collectionId, request.trail), email))
 
-  private def sendAlert(trail: ClientHydratedTrail, email: String, collectionId: String): Future[Option[String]] = {
-    def handleSuccessfulFuture(result: Either[ApiClientError, Unit]) = result match {
-      case Left(error) =>
-        structuredLogger.putLog(LogUpdate(HandlingBreakingNewsCollection(collectionId), email), "error", Some(new Exception(error.description)))
-        Some(error.description)
-      case Right(_) => None
-    }
-    def withExceptionHandling(block: => Future[Option[String]]): Future[Option[String]] = {
-      Try(block) match {
-        case Success(futureMaybeError) => futureMaybeError
-        case Failure(t: Throwable) =>
-          val message = s"Exception in breaking news client send for trail ${trail.headline} because ${t.getMessage}"
-          logger.error(message, t)
-          Future.successful(Some(message))}
+      if(request.trail.alert.getOrElse(false)) {
+        client.send(request).map(request -> _)
+      } else {
+        // TODO MRB: use custom BreakingNewsError enum
+        logger.error(s"Failed to send a breaking news alert for trail ${request.trail} because alert was missing")
```

Comment:
We could use the structured logger as on line 50 to make the logs more discoverable